README.md

```diff
@@ -31,7 +31,7 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Node12-x:<VERSION>
 # OR
 arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Node10-x:<VERSION>
 # OR
-arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Node8-10:<VERSION>
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Node8-10:<VERSION> # (Deprecated)
```

Comment:
Given that Node 8 functions can't be updated (and therefore no one new can install the Node 8 layer), does it make sense to just delete this?

Reply:
I think we should leave it there, just to make it clear to anyone upgrading there old lambdas to know what to search for. Maybe we can remove it in a few months?